### PR TITLE
Document the RAVEN_URL environment variable

### PR DESCRIPTION
**Pull request by rowanmanning**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ We configure Origami Image Service using environment variables. In development,
   * `CLOUDINARY_ACCOUNT_NAME`: The name of the Cloudinary account to use in image transforms.
   * `IMGIX_SECURE_URL_TOKEN`: The Imgix secure URL token to use in image transforms.
   * `IMGIX_SOURCE_NAME`: The Imgix source name to use in image transforms.
+  * `RAVEN_URL`: The Sentry URL to send error information to.
```
Comment (onishiweb) [inline README.md]: Raven vs. Sentry? Should they be given the same name?

Comment (rowanmanning) [inline README.md]: Ideally probably, but this is built into n-express and that expects `RAVEN_URL`. I might PR a change to n-express some time in the future.

Comment (onishiweb) [inline README.md]: Ok cool, good to know.

**Comment by onishiweb** [comment]
👍

**Pull request merged**